_posts/2022-12-13-release.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Release 2022.50
+author: Priya
+date: 2022-12-13
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Show original column names.
+If a column is renamed, users may have difficulty finding its original name. To help with this, Mammoth now shows the original name in the column header when the user hovers over it.
+
+### Defect fixes
+
+* Apply filter - Second column name does not appear in function panel and task description.
+* Navbar is not fixed in settings page.It moves when we scroll. 
+* Publish rerun request never finishes.
+* Batch column section in preview panel - Cancel button should not overlap the text.
+* File upload model- Close icon shows incorrect tooltip message.
+
+### Internal Improvements
+* The user events field that records browser details previously displayed incorrect information for Brave users. This issue has now been resolved.
```

Comment:
Lets remove this statement. Also remove the section Internal Improvements

Reply:
removed the section and the statement